content/_data/changelogs/weekly.yml

```diff
@@ -1934,6 +1934,62 @@
     # pull: 3247 too minor (test only)
     # pull: 3239 remove obsolete test
     # pull: 3236 too minor (imports and Javadoc in test)
+- version: "2.104"
+  date: 2018-01-28
+  changes:
+    - type: rfe
+      message: >
+        Whitelist additional safe Java platform types for use in XStream (XML serialization) and remoting (agent communication).
+      references:
+        - pull: 3251
+        - pull: 3252
+        - pull: 3253 # this one has no issue, so add all to references
```

Comment:
URL to https://github.com/KostyaSha/github-integration-plugin/issues/253 ?

Reply:
Not the Jenkins project issue tracker.